src/redux/books/books.js

```diff
@@ -0,0 +1,37 @@
+// Action Types
+const ADD_BOOK = 'ADD';
+const REMOVE_BOOK = 'REMOVE';
+
+// Default state
+const defaultState = [];
+
+// Reducer function
+const booksReducer = (state = defaultState, action) => {
```

Comment:
- [ ] [OPTIONAL] As shown in the Demo video below, your Add Book button does not add a book yet. This could be because you haven't created a store and you're not also using the store to get the data from redux to your application. After creating your store in your configureStore.js file. please remember to surround your index.js with the provider and set the value to the store from the configureStore.js file(after importing it)

## EXAMPLE
```
import store from './redux/configureStore';

ReactDOM.render(
  <React.StrictMode>
    <Provider store={store}>
      <App />
    </Provider>
    ,
  </React.StrictMode>,
  document.getElementById('root'),
);

```

https://user-images.githubusercontent.com/52934128/167868514-d2507a14-17c8-4c66-854c-c07a8daa931c.mp4